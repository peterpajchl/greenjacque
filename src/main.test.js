import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { shallow } from 'enzyme';
import Main from './main';

Enzyme.configure({ adapter: new Adapter() });

describe('<Main />', () => {

  const container = shallow(<Main />);

  it('should match the snapshot', () => {
    expect(container.html()).toMatchSnapshot();
  });

});