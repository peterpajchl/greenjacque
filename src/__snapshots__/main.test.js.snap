// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should match the snapshot 1`] = `"<header class=\\"main-header\\"><div class=\\"main-header__title\\">GreenJacque</div><nav class=\\"main-nav\\"><ul class=\\"main-nav__list\\"><li class=\\"nav-list-item\\"><button class=\\"btn\\" type=\\"button\\">Save</button></li><li class=\\"nav-list-item\\"><button class=\\"btn btn--caution\\" type=\\"button\\">Reset</button></li></ul></nav></header><main class=\\"main-body\\"><div class=\\"notification notification--info \\"><p class=\\"notification__text\\"></p></div><div class=\\"main-body__block main-body__block--deck\\"><div class=\\"body-header\\"><h3 class=\\"body-header__title\\">Deck</h3><div class=\\"body-header__counter\\">0/52</div></div><ul class=\\"cards-in-deck__list\\"></ul><div class=\\"card-controls\\"><button class=\\"btn control\\" type=\\"button\\">Shuffle</button><form method=\\"GET\\" action=\\"/\\" class=\\"draw-card control\\"><label for=\\"draw-card-input\\" class=\\"sr-only\\">Enter number of cards to draw</label><input type=\\"number\\" id=\\"draw-card-input\\" class=\\"draw-card__input\\" min=\\"1\\" max=\\"52\\" value=\\"1\\"/><button class=\\"draw-card__submit btn\\" type=\\"button\\">Draw</button></form></div></div><div class=\\"main-body__block main-body__block--hand\\"><div class=\\"body-header\\"><h3 class=\\"body-header__title\\">Hand</h3><div class=\\"body-header__counter\\">0 (0)</div></div><ul class=\\"cards-in-hand__list\\"></ul><div class=\\"card-controls\\"><button class=\\"btn control\\" type=\\"button\\" disabled=\\"\\">Sort</button></div></div></main>"`;
